From 2e265e85777345a318084c2f1f3b684f7e7ff4a4 Mon Sep 17 00:00:00 2001
From: Cheng C Yang <cheng.c.yang@intel.com>
Date: Thu, 28 Mar 2019 18:06:54 +0800
Subject: [PATCH] Change some properties name in SOL Dbus

Change some properties name in SOL D-Bus interface to make the meaning
of the properties more clearly.

Signed-off-by: Cheng C Yang <cheng.c.yang@intel.com>
---
 xyz/openbmc_project/Ipmi/SOL.interface.yaml | 25 +++++++++++++++++--------
 1 file changed, 17 insertions(+), 8 deletions(-)

diff --git a/xyz/openbmc_project/Ipmi/SOL.interface.yaml b/xyz/openbmc_project/Ipmi/SOL.interface.yaml
index 94db59f..96c8c87 100644
--- a/xyz/openbmc_project/Ipmi/SOL.interface.yaml
+++ b/xyz/openbmc_project/Ipmi/SOL.interface.yaml
@@ -2,8 +2,8 @@ description: >
     SOL properties use for Get/Set SOL config parameter command in host-ipmid
     sending config to SOL process in net-ipmid.
     Since some platforms need to access Get/Set SOL config parameter command
-    through KCS, and current sol manager is implemented in net-ipmid and
-    cannot be accessed by host-ipmid, add a dbus interface for host-ipmid
+    through KCS, and current SOL manager is implemented in net-ipmid and
+    cannot be accessed by host-ipmid, add a D-Bus interface for host-ipmid
     command to transfer properties to net-ipmid.
     This interface will be implemented in phosphor-settings.
 properties:
@@ -17,12 +17,20 @@ properties:
       description: >
         SOL Enable property, this controls whether the SOL payload type
         can be activated.
-    - name: Authentication
+    - name: ForceEncryption
+      type: boolean
+      description: >
+        If SOL enable Force Payload Encryption.
+    - name: ForceAuthentication
+      type: boolean
+      description: >
+        If SOL enable Force Payload Authentication
+    - name: Privilege
       type: byte
       description: >
-        If SOL enable Force Payload Encryption and Authenticaton.
-        And the minimun operating privilege level SOL required.
-    - name: Accumulate
+        Sets the minimum operating privilege level that is required to
+        be able to activate SOL by Activate Payload command.
+    - name: AccumulateIntervalMS
       type: byte
       description: >
         Character Accumulate Interval in 5ms increments.
@@ -31,13 +39,14 @@ properties:
       type: byte
       description: >
         BMC will automatically send an SOL character data packet containing
-        this number of characters.
+        this number of characters as soon as this number of characters
+        (or greater) has been accepted from the baseboard serial controller.
     - name: RetryCount
       type: byte
       description: >
         Packet will be dropped if no ACK/NACK received by time retries
         expire.
-    - name: RetryInterval
+    - name: RetryIntervalMS
       type: byte
       description: >
         Retry Interval in 10ms increments.
-- 
2.16.2

